shell/platform/embedder/tests/embedder_test_context.h

```diff
@@ -79,6 +79,8 @@ class EmbedderTestContext {
 
   size_t GetSoftwareSurfacePresentCount() const;
 
+  std::vector<FlutterFrameInfo> GetGLFBOFrameInfos();
```

Comment:
Please add a declaration comment, since the implementation isn't inline to make it obvious what it does. (Or, alternately, comment the ivar.)

Reply:
Done